src/components/modals/WalletListModal.tsx

```diff
@@ -157,7 +157,7 @@ export function WalletListModal(props: Props) {
         title={s.strings.deposit_to_bank}
         message={sprintf(s.strings.wallet_list_modal_confirm_s_bank_withdrawal, config.appName)}
         buttons={{
-          continue: { label: s.strings.legacy_address_modal_continue }
+          continue: { label: s.strings.continue_fragment }
```

Comment:
Do not rename string keys even if they seem inappropriate. This will cause a re-translation to be needed.